docs/guidance/community_engagement.md

```diff
@@ -10,9 +10,11 @@ Keeping your community engaged is important if you want to keep contributions co
 
 There are many options to choose from when it comes to community platforms. Two recommended platforms are:
 
-- [GitHub Discussions](https://docs.github.com/en/discussions) takes the least effort to setup if your project repository is on GitHub.
-- [Discourse](https://www.discourse.org/) is an open source platform that provides various features for community discussions and tools for moderation. 
+- [GitHub Discussions](https://docs.github.com/en/discussions) takes the least effort to setup if your project repository is on GitHub.However, according to users GitHub is mainly used for technical developments rather than as a forum.
```

Comment:
```suggestion
- [GitHub Discussions](https://docs.github.com/en/discussions) takes the least effort to setup if your project repository is on GitHub. However, according to many users GitHub is mainly used for technical development work rather than as a forum.
```